README.md

```diff
@@ -66,7 +66,7 @@ Usage: npm <command> blah blah...
 ```
 
 ## One more thing
-You can notice good news when you didn't know that npm latest version was updated.
+npmbrew uses [update-notifier](https://github.com/yeoman/update-notifier/), to let users know when there is a new version available for download.
```

Comment:
npmbrew use update-notifier for oneself. Display update notifier for npm by [this func](https://github.com/watilde/npmbrew/blob/master/lib/modules/npm.js#L44-L64). So could you changed this description only a little?